s3backgrounddelete/s3backgrounddelete/object_recovery_processor.py

```diff
@@ -50,29 +50,16 @@ def consume(self):
         """Consume the objects from object recovery queue."""
         self.server = None
         try:
-            #Conditionally importing ObjectRecoveryRabbitMq/ObjectRecoveryMsgbusConsumer when config setting says so.
+            #Conditionally importing ObjectRecoveryMsgbusConsumer when config setting says so.
             if self.config.get_messaging_platform() == MESSAGE_BUS:
                 from s3backgrounddelete.object_recovery_msgbus import ObjectRecoveryMsgbus
 
                 self.server = ObjectRecoveryMsgbus(
                     self.config,
                     self.logger)
-            elif self.config.get_messaging_platform() == RABBIT_MQ:
-                from s3backgrounddelete.object_recovery_queue import ObjectRecoveryRabbitMq
-
-                self.server = ObjectRecoveryRabbitMq(
-                    self.config,
-                    self.config.get_rabbitmq_username(),
-                    self.config.get_rabbitmq_password(),
-                    self.config.get_rabbitmq_host(),
-                    self.config.get_rabbitmq_exchange(),
-                    self.config.get_rabbitmq_queue_name(),
-                    self.config.get_rabbitmq_mode(),
-                    self.config.get_rabbitmq_durable(),
-                    self.logger)
             else:
                 self.logger.error(
-                "Invalid argument specified in messaging_platform use message_bus or rabbit_mq")
+                "Invalid argument specified in messaging_platform use 'message_bus'")
```

Comment:
log the value as well, and do not say to use 'message_bus'. Tomorrow we may use something else... just log the value and say that this value is not supported.

Reply:
done.